components/tfx/Evaluator/with_URI_IO/component.py

```diff
@@ -0,0 +1,103 @@
+# flake8: noqa
+
+from typing import NamedTuple
+
+def Evaluator(
+    examples_uri: 'ExamplesUri',
+    model_uri: 'ModelUri',
+    output_evaluation_uri: 'ModelEvaluationUri',
+    #output_blessing_uri: 'ModelBlessingUri',
```

Comment:
Maybe leave a TODO here to implement this when optional output is ready.

I still feel like Pusher cannot be done in the current situation. Do we have a work-around to make Pusher work without `ModelBlessing`?

Reply:
I've taken another look and I see that I can actually enable this output for the URI-IO-based component (since the system is not following that URI).
Uncommented this output.